capsules/classifier_safety_gear_openvino/README.md

```diff
@@ -1,9 +1,10 @@
 # About the Capsule
 ## Usage
-This capsule is for detecting if a person wears safety gears (safety vest and safety hat).
+This capsule is for classifying if a person wears safety equipments 
+(safety vest and safety hat).
```

Comment:
I think the readme should make it more clear what classification means. Is it `yes/no`? What are the requirements?